dashboard/templates/layouts/base.html

```diff
@@ -53,6 +53,11 @@
           <a class="nav-link" href="/extra">{{ config['EXTRA_TAB'] }}</a>
         </li>
         {% endif %}
+        {% if config['COLUMN_DESCRIPTION_ACTIVE'] %}
```

Comment:
It would be good to have consistent naming: "column description" vs "table descriptions"